codejam-canvas/32x32.js

```diff
@@ -0,0 +1,6226 @@
+        255
+    ],
+    [
+        145,
+        49,
+        40,
+        255
+    ],
+    [
+        143,
+        49,
+        40,
+        255
+    ],
+    [
+        134,
+        49,
+        41,
+        255
+    ],
+    [
+        146,
+        49,
+        40,
+        255
+    ],
+    [
+        170,
+        49,
+        37,
+        255
+    ],
+    [
+        157,
+        49,
+        39,
+        255
+    ],
+    [
+        161,
+        49,
+        38,
+        255
+    ],
+    [
+        200,
+        48,
+        35,
+        221
+    ],
+    [
+        213,
+        50,
+        35,
+        103
+    ]
+    ],
+    [
+    [
+        205,
+        50,
+        33,
+        122
+    ],
+    [
+        210,
+        49,
+        33,
+        217
+    ],
+    [
+        184,
+        49,
+        36,
+        235
+    ],
+    [
+        160,
+        49,
+        38,
+        252
+    ],
+    [
+        155,
+        49,
+        39,
+        245
+    ],
+    [
+        163,
+        49,
+        38,
+        255
+    ],
+    [
+        157,
+        49,
+        39,
+        253
+    ],
+    [
+        157,
+        49,
+        39,
+        248
+    ],
+    [
+        158,
+        49,
+        39,
+        255
+    ],
+    [
+        161,
+        49,
+        38,
+        251
+    ],
+    [
+        157,
+        49,
+        39,
+        249
+    ],
+    [
+        153,
+        49,
+        40,
+        248
+    ],
+    [
+        156,
+        49,
+        39,
+        245
+    ],
+    [
+        168,
+        49,
+        39,
+        245
+    ],
+    [
+        154,
+        49,
+        39,
+        248
+    ],
+    [
+        155,
+        49,
+        39,
+        243
+    ],
+    [
+        136,
+        49,
+        41,
+        252
+    ],
+    [
+        155,
+        49,
+        39,
+        254
+    ],
+    [
+        191,
+        49,
+        35,
+        252
+    ],
+    [
+        189,
+        49,
+        35,
+        252
+    ],
+    [
+        175,
+        49,
+        37,
+        255
+    ],
+    [
+        156,
+        49,
+        39,
+        244
+    ],
+    [
+        145,
+        49,
+        40,
+        248
+    ],
+    [
+        159,
+        49,
+        39,
+        248
+    ],
+    [
+        151,
+        49,
+        39,
+        247
+    ],
+    [
+        156,
+        49,
+        39,
+        252
+    ],
+    [
+        150,
+        49,
+        39,
+        250
+    ],
+    [
+        166,
+        49,
+        38,
+        249
+    ],
+    [
+        166,
+        49,
+        38,
+        255
+    ],
+    [
+        163,
+        49,
+        38,
+        255
+    ],
+    [
+        202,
+        49,
+        34,
+        252
+    ],
+    [
+        212,
+        51,
+        33,
+        131
+    ]
+    ],
+    [
+    [
+        206,
+        49,
+        34,
+        67
+    ],
+    [
+        204,
+        50,
+        34,
+        180
+    ],
+    [
+        205
```

Comment:
use array `join` method to create rgba string
```suggestion
        ctx.fillStyle = `rgba(${A[row][col].join(',')})`; 
```